cmd/simulacra/simulacra.go

```diff
@@ -181,16 +194,16 @@ func getInstanceName() string {
 
 func getConfigFromYaml(configPath string) (*Config, error) {
 	var config Config
-	file, err := ioutil.ReadFile(configPath)
+	file, err := os.ReadFile(configPath)
 	if err != nil {
 		return nil, err
 	}
 	if err := yaml.Unmarshal(file, &config); err != nil {
 		return nil, err
 	}
 
-	if config.Platform == "" {
-		config.Platform = defaultPlatform
+	if config.ImageFamily == "" && config.Image == "" {
```

Comment:
So `ImageFamily` is okay to be empty if `Image` is set?

Reply:
yep, it's an either or. If both are set, there's actually an error we send from the integration testing side.